### ISSUE TITLE
Warnings building wasm crate.

### ISSUE THREAD

**Issue by cthulahoops** [RustPython/RustPython#681]
```cargo test --all``` gives:

```
warning: unused import: `JsCast`
  --> wasm/lib/src/vm_class.rs:14:32
   |
14 | use wasm_bindgen::{prelude::*, JsCast};
   |                                ^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

warning: unused import: `JsCast`
  --> wasm/lib/src/vm_class.rs:14:32
   |
14 | use wasm_bindgen::{prelude::*, JsCast};
   |                                ^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

warning: unused `std::result::Result` that must be used
warning: unused `std::result::Result` that must be used
  --> wasm/lib/src/convert.rs:50:25
   |
50 |                         Reflect::set(&js_err, &"row".into(), &lineno.into());
   |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  --> wasm/lib/src/convert.rs:50:25
   |
50 |                         Reflect::set(&js_err, &"row".into(), &lineno.into());
   |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_must_use)] on by default
   = note: this `Result` may be an `Err` variant, which should be handled

   |
   = note: #[warn(unused_must_use)] on by default
   = note: this `Result` may be an `Err` variant, which should be handled
```

**Comment by coolreader18** [comment]
The second one I fixed with #630, the first one should be pretty trivial.

**Comment by coolreader18** [comment]
This is fixed now.
